cnxarchive/database.py

```diff
@@ -484,7 +487,8 @@ def republish_module(td, plpy):
     # Module is republished
     for collection_id in get_collections(current_module_ident, plpy):
         minor = next_version(collection_id, plpy)
-        new_ident = republish_collection(minor, collection_id, plpy)
+        new_ident = republish_collection(submitter, submitlog, minor,
```

Comment:
This is the part that is not covered by the tests you modified.  I wanted to make sure that the correct `submitter` and `submitlog` are being used in the collection minor version.